regulations/static/regulations/js/source/views/header/preamble-head-view.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
```

Comment:
Eep, sorry, I meant a code comment. We've got... 4(ish?) views related to comments now -- it'd be a good idea to document what they are for.